internal/pkg/table/path.go

```diff
@@ -144,6 +144,9 @@ type Path struct {
 	// For BGP Nexthop Tracking, this field shows if nexthop is invalidated by IGP.
 	IsNexthopInvalid bool
 	IsWithdraw       bool
+
+	RejectedByPolicy  string
+	AcceptedByPolicy  string
 }
```

Comment:
Path object is immutable. Once it's inserted to the table, it can't be changed.

Reply:
I'm aware of the hackyness of the PR ;) - it's just prove of concept that it would work